README.md

```diff
@@ -1 +1,159 @@
+                    "type": "object",
+                    "properties": {
+                    "value": {
+                        "title": "Value",
+                        "description": "A description of block attr.",
+                        "type": "string",
+                        "format": "date-time"
+                    }
+                    },
+                    "required": [
+                    "value"
+                    ],
+                    "block_type_slug": "block-slug",
+                    "secret_fields": [],
+                    "block_schema_references": {}
+                },
+                "capabilities": [],
+                "version": "2.7.12"
+                }
+            }
+        }
+    }
+```
+
+### Collection Metadata
+```json
+{
+    "collection-name": {
+        "name": "collection-name",
+        "author": "Prefect Technologies, Inc.",
+        "latest_version": "0.2.0",
+        "downloads": {
+            "last_day": -1,
+            "last_month": -1,
+            "last_week": -1
+        },
+        "keywords": "prefect",
+        "home_page": "https://github.com/PrefectHQ/collection-name",
+        "package_url": "https://pypi.org/project/collection-name/",
+        "requires_dist": [
+            "prefect (>=2.0.0)",
+            ...
+        ],
+        "requires_python": ">=3.7"
+    }
+}
+```
+
+### Flow Metadata
+```json
+{
+    "prefect-collection-that-defines-flow": {
+        "some-prefect-flow": {
+            "name": "some-prefect-flow",
+            "parameters": {
+                "title": "Parameters",
+                "type": "object",
+                "properties": {
+                    "flow_parameter": {
+                        "title": "flow_parameter",
+                        "description": "Description of flow parameter.",
+                        "position": 0,
+                        "allOf": [
+                            {
+                                "$ref": "#/definitions/FlowParameter"
+                            }
+                        ]
+                    }
+                },
+                "required": [
+                    "flow_parameter"
+                ],
+                "definitions": { ... Schema of flow parameter, if applicable... }
+            },
+            "install_command": "pip install prefect-collection",
+            "description": {
+                "summary": "First line of the docstring",
+                "returns": "Description of the return value per docstring",
+                "examples": [
+                    "from prefect_collection import flow\nflow()",
+                ]
+            }
+        }
+    }
+}
+```
+
+## Development
+Setup a virtual environment and install the dependencies:
+
+```bash
+python3 -m venv venv
+source venv/bin/activate # or your preferred method of activating a venv
+pip install .
+```
+
+Run the tests:
+```bash
+pytest
+```
```

Comment:
I think there's more to this (e.g. creating github token and creating the deployments).